src/main/java/eiteam/esteemedinnovation/boiler/TileEntityBoiler.java

```diff
@@ -68,48 +77,51 @@ public static int getItemBurnTime(@Nonnull ItemStack stack) {
             return 0;
         }
         Item item = stack.getItem();
-
+        int burnTime = ForgeEventFactory.getItemBurnTime(stack);
+        if (burnTime >= 0) {
+            return burnTime;
+        }
         if (item instanceof ItemBlock && Block.getBlockFromItem(item) != Blocks.AIR) {
             Block block = Block.getBlockFromItem(item);
 
             if (OreDictHelper.slabWoods.contains(item)) {
```

Comment:
Regarding these changes `return` -> `burnTime =`, I'm curious your rationale. It seems that it would be faster to return quick than to check a bunch of additional conditionals, no?